src/sourmash/commands.py

```diff
@@ -646,10 +645,9 @@ def gather(args):
         error('query signature needs to be created with --scaled')
         sys.exit(-1)
 
-    # downsample if requested
     if args.scaled:
         notify('downsampling query from scaled={} to {}',
-               query.minhash.scaled, int(args.scaled))
```

Comment:
please indent to same level as open parenthesis

Reply:
same?

Reply:
I already de-indented it actually.

Reply:
when I look at it, I see 
<img width="619" alt="Screen Shot 2021-07-16 at 12 38 36 PM" src="https://user-images.githubusercontent.com/51016/126000266-eab48fb9-a52b-449c-a345-d1375e0f25d0.png">
- the second line should start at the `(` of the line above.

Reply:
oh, i see, i'll fix that